admin/spec/requests/solidus_admin/adjustment_reasons_spec.rb

```diff
@@ -64,8 +64,13 @@
   end
 
   describe "GET /edit" do
-    it "renders the edit template with a 200 OK status" do
+    it "redirects when the request is not Turbo-Frame" do
```

Comment:
No, we want that these routes still accessible in ie. new tabs or tests w/o javascript.